### ISSUE TITLE
unnecessary Makefile.PL

### ISSUE THREAD

**Issue by szabgab** [patjoly/geo-tcx#1]
As this distribution uses Dist::Zilla, I belive it is unnecessary to have a Makefile.PL in the repository.
It also seems to cause problems:

```
$ dzil test
[DZ] building distribution under .build/KWtstz9M9b for installation
[DZ] beginning to build Geo-TCX
[DZ] guessing dist's main_module is lib/Geo/TCX.pm
[VersionFromModule] dist version 1.01 (from lib/Geo/TCX.pm)
[DZ] attempt to add Makefile.PL multiple times; added by: filename set by GatherDir (Dist::Zilla::Plugin::GatherDir line 235); encoded_content added by @Basic/GatherDir (Dist::Zilla::Plugin::GatherDir line 236); content set by MakeMaker (Dist::Zilla::Plugin::MakeMaker line 337); content added by @Basic/MakeMaker (Dist::Zilla::Plugin::MakeMaker line 152)
aborting; duplicate files would be produced at /home/gabor/perl5/lib/perl5/Dist/Zilla/App/Command/test.pm line 82.
```

**Comment by patjoly** [comment]
I am a bit torn on this one. First off, I just migrated the dist management to dzil this week-end and am new to this very useful tool. I tried to mimick a few other repos that use dzil such as Mechanize and they keep a Makefile.PL in the top-level folder and maintain it at each version.

From a selfish perspective, I need to have a Makefile.PL handy at all times because i run, from vim :! cd ..; make && perl -Mblib -d "t/%" dozens and dozens of time when I am coding (with a vim mapping). I just delete it before building/releasing with dzil.

But from a user's perspective, I think it may be a bit onerous to ask them to install Dist::Zilla and all the required Plugins if they clone or fork the repo, so they can use the usual make && make test; make install after generating the Makefile.

That said, I certainly would be happy to get rid of a file in the repo, so could just add it to .gitignore.

I am thinking about it and welcome your perspective. Thank you.
